smartseq2_single_sample/ss2_single_sample.wdl

```diff
@@ -0,0 +1,181 @@
+task Star {
+  File input_fastq_read1
+  File input_fastq_read2
+  File gtf
+  File star_genome
+  
+  command {
+    tar -zxvf ${star_genome}
+    STAR  --readFilesIn ${input_fastq_read1} ${input_fastq_read2} --genomeDir ./star  --quantMode TranscriptomeSAM   --outSAMstrandField intronMotif --genomeLoad NoSharedMemory --sjdbGTFfile ${gtf} --readFilesCommand "zcat" --twopassMode Basic --outSAMtype BAM SortedByCoordinate  --outSAMunmapped Within --limitBAMsortRAM 30000000000 
```

Comment:
One other thing we typically do, formatting wise, is put parameters each on their own line for ease of reading. Here is an example from another WDL:
```
command {
    java -Dsamjdk.compression_level=${compression_level} -Xms128m -jar /usr/gitc/picard.jar \
      CollectQualityYieldMetrics \
      INPUT=${input_bam} \
      OQ=true \
      OUTPUT=${metrics_filename}
  }
```

Reply:
I see now that you did this in tasks below, so copying that format would work here :)